src/components/card/z-card-info/index.tsx

```diff
@@ -61,18 +69,48 @@ export class zCardInfo {
 
   renderGeneralSection() {
     return (
-      <section>
-        Autore: <b>{this?.cardData?.author}</b>
-        <br />
-        Anno: <b>{this?.cardData?.year}</b>
-        <br />
+      <section
+        class={`info-wrapper ${this.hiddenContent ? "hidden" : ""}`}
+        onClick={() => {
+          if (this.hiddenContent) this.tooltip = !this.tooltip;
+        }}
+        ref={(el) => (this.infoWrapper = el)}
+      >
+        {this?.cardData?.author ? (
```

Comment:
questa parte è poco leggibile, valuta se mettere i vari rendering in metodi dedicati

Reply:
Ci avevo pensato. Okie.